apps/prairielearn/assets/scripts/lib/countdown.ts

```diff
@@ -59,7 +59,7 @@ export function setupCountdown(options: {
   let serverTimeLimitMS: number;
   let clientStart: number;
   let updateServerIfExpired = true;
-  let nextCountdownDisplay: number | null = null;
+  let nextCountdownDisplay: ReturnType<typeof setTimeout> | null = null;
```

Comment:
I'm not sure why this is causing an error now but wasn't before.

Reply:
I ran into this too in #8617